.github/workflows/atlas-migrations.yml

```
@@ -24,207 +24,266 @@
+        id: shadow-db
+        run: |
+          # Get the container IP
+          DB_HOST=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' torus-ts-postgres-1)
+
+          # Use the database URL in your application
+          DATABASE_URL="postgresql://postgres:postgres@${DB_HOST}:5432/torus-ts-db?sslmode=disable"
+
+          echo "db_url=${DATABASE_URL}" >> $GITHUB_OUTPUT
+
+          echo "Waiting for PostgreSQL to be ready..."
+          until pg_isready -d "${DATABASE_URL}"; do 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)